src/main.rs

```diff
@@ -12,21 +12,15 @@ use std::str::FromStr;
 type AppId = str;
 type DeviceId = str;
 
+const NO_COMMAND_MESSAGE: &str = "No subcommand supplied, should be `app` or `device`. Run `drgctl help` to see available commands.";
```

Comment:
I think this const should be in  the `Arguments` module

Reply:
This string is not required anymore. Thanks